src/collectors/httpjson/httpjson.py

```diff
@@ -27,7 +28,8 @@ def get_default_config(self):
         default_config = super(HTTPJSONCollector, self).get_default_config()
         default_config.update({
             'path': 'httpjson',
-            'url': 'http://localhost/stat'
+            'url': 'http://localhost/stat',
+            'headers': {'User-Agent': 'Diamond HTTP collector'},
```

Comment:
If you set 

```

[[HTTPJSONCollector]]
enabled = True
measure_collector_time = False
url = http://localhost/HelloWorld
[[[headers]]]
X-Hello=World 
```
it should work as expected (set a header key: X-Hello and value World)